EnglishFilter/sections/foreign.txt

```diff
@@ -2024,6 +2024,9 @@ hardverapro.hu##.uad-details > .col-md-4.text-center > div[class*=" "]:has(> h4
 ! https://github.com/AdguardTeam/AdguardFilters/issues/31163
 figyelo.hu#$#div[class$="__interstitial_active"] { display: none!important; }
 figyelo.hu#$#div[class$="__interstitial_active"] + .page__body { display: block!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/36387
+pi-shop.hu#$##mobile-nav { top:0px!important; }
+pi-shop.hu#$#body.aurora-nanobar-active { padding-top:0px!important; }
```

Comment:
That rule pack is meant for cookies as it is only broken when cookie message is removed.